<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PWSZ Prover9 Insert</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QTextEdit" name="formula">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="input">
        <property name="text">
         <string>Input</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QTextBrowser" name="proverOutput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Formula:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextBrowser" name="proverInput"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="showExample">
          <property name="text">
           <string>Show Example</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="runProver">
          <property name="text">
           <string>Run Prover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="onlyProof">
          <property name="text">
           <string>Proof only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="makeInputFile">
        <property name="text">
         <string>Make Input File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>formula</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>proverInput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>proverOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
